api/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    #participant
+    path('register/', views.RegisterView.as_view(), name='participant-register'),
+    path('auth/', views.ParticipantAuthView.as_view(), name='auth-participant'),
+    path('delete_participant/<int:id>/', views.ParticipantDeleteView.as_view(), name='delete_participant'),
+    path('logout/', views.LogoutView.as_view(), name='logout'),
+    #team
+    path('create_team/', views.TeamCreateView.as_view(), name='create_team'),
+    path('join_team/<int:id>/', views.ParticipantJoinTeamView.as_view(), name='join_team'),
```

Comment:
use kebab case instead of snake case in your endpoints

Reply:
you mean one below the other ?